Maple2.File.Ingest/Mapper/ItemMapper.cs

```diff
@@ -70,6 +72,48 @@ public class ItemMapper : TypeMapper<ItemMetadata> {
                 expirationDuration = data.life.usePeriod;
             }
 
+            var hairList = new List<DefaultHairMetadata>();
+            // parse default hair positions
+            foreach (Slot slot in data.slots.slot.Where(dataSlots => dataSlots.name == "HR")) {
```

Comment:
do you know how `DefaultHairs` is going to be used? Trying to see if there's a more generic way to parse+store it.

Reply:
It's used in the random hair npc.
It would pick one available entry for hair positioning and then a value between min and max scale for hair length.

Reply:
Seems like there is only ever 1 entry in this list, do you know of a case with multiple?

Reply:
For example, 10200010 has multiple entries

Reply:
I see, looks like it's 1 per pigtail, I guess this is fine.

```xml
<slot name="HR">
  <asset name="Data/Resource/Model/Item/0/02/00200010_F_BasicHair02_A.nif" selfnode="HR" targetnode="HR" attachnode="" pony="0" zalign="0" replace="1" earfold="0" ani="0" emotionhide="0" gender="2" capscale="1" weapon="0" placeable="0">
    <physx action="" />
  </asset>
  <asset name="urn:gamebryo-animation:urn:gamebryo-animation:urn:00200010_F_PiPi_P_A" selfnode="Point01" targetnode="Bip01 Head" attachnode="" pony="1" zalign="0" replace="0" earfold="0" ani="0" emotionhide="0" gender="2" capscale="1" weapon="0" placeable="180">
    <custom icon="./Data/Resource/Image/Item/icon/customize/position/10200010_1.png" position="55.850037, -4.049072, -18.901474" rotation="-8.056290, -40.748642, 1.041489" />
    <custom icon="./Data/Resource/Image/Item/icon/customize/position/10200010_2.png" position="34.459137, -11.702539, -28.219086" rotation="8.431839, -82.585579, 5.826733" />
    <custom icon="./Data/Resource/Image/Item/icon/customize/position/10200010_3.png" position="18.347984, -22.655460, -21.698593" rotation="-138.176010, -68.960564, -179.196945" />
    <physx action="" />
  </asset>
  <asset name="urn:gamebryo-animation:urn:gamebryo-animation:urn:00200010_F_PiPi_P2_A" selfnode="Point01" targetnode="Bip01 Head" attachnode="" pony="1" zalign="0" replace="0" earfold="0" ani="0" emotionhide="0" gender="2" capscale="1" weapon="0" placeable="180">
    <custom icon="" position="54.642906, -4.643814, 21.399384" rotation="-173.230316, -44.148136, -1.642636" />
    <custom icon="" position="36.088089, -11.924242, 25.931381" rotation="173.847427, -79.176346, -8.600166" />
    <custom icon="" position="19.429283, -22.765934, 20.766247" rotation="-47.793465, -71.031036, -174.788925" />
    <physx action="" />
  </asset>
  <scale value="0.800000,1.000000,1.200000" min="0.8" max="1.2" reverse="0" />
  <decal />
</slot>
```